rootfs/usr/bin/gnome-session-oneshot

```diff
@@ -5,12 +5,41 @@ set -e
 die() { echo >&2 "!! $*"; exit 1; }
 
 SENTINEL_FILE="steamos-session-select"
+RETURN_SESSION_FILE="chimeraos-session-return"
```

Comment:
What is this?

Reply:
A file Introduced to keep the name of the session we should return to. But I most likely screwed up it's location on write